CHANGELOG.md

```diff
@@ -8,8 +8,9 @@
 
 ##### Bug Fixes
 
-* None.  
-
+* Fix regression resulting in projecs existing product ref groups being altered
```

Comment:
Please add two spaces to the end of the changelog entry (i.e `* Fix regression resulting in projecs existing product ref groups being altered  `)